HonestHouse/HonestHouse/Sources/Core/Network/CCAPI/4.13. Camera Status/CameraStatusAPI.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  CameraStatusAPI.swift
+//  HonestHouse
+//
+//  Created by Rama on 11/10/25.
+//
+
+import Foundation
+
+enum TimeoutType: String {
+    case immediately = "immediately"
+    case short = "short"
+    case long = "long"
+}
+
+enum CameraStatusAPI {
+    case getPolling(TimeoutType)
+    
+    var endpoint: String {
+        switch self {
+        case .getPolling:
+            return "event/polling"
+        }
+    }
+    
+    func path(with version: VersionType) -> String {
+        return "\(version.description)/\(endpoint)"
+    }
+}
```

Comment:
`TimeoutType`이 현재 `CameraStatusAPI`에서만 사용되고 있다면 내부로 이동하는 것을 제안드립니다~~(네임스페이스를 명확히 하기 위함.)

```suggestion
enum CameraStatusAPI {
    enum TimeoutType: String {
        case immediately = "immediately"
        case short = "short"
        case long = "long"
    }
    
    case getPolling(TimeoutType)
    
    var endpoint: String {
        switch self {
        case .getPolling: return "event/polling"
        }
    }
    
    func path(with version: VersionType) -> String {
        return "\(version.description)/\(endpoint)"
    }
}
```